src/http2_stream.erl

```diff
@@ -838,8 +838,8 @@ recv_frame({_FH=#frame_header{
      }, ConnectionState};
 
 recv_frame(F, {S,_C}=Acc) ->
-    lager:info("RecvF: ~p", [F]),
-    lager:info("RecvS: ~p", [S]),
+    lager:debug("RecvF: ~p", [F]),
```

Comment:
I was actually going to drop these entirely. this module could use some cleanup now that it's working. I'll get to it.